pkg/queue/breaker.go

```diff
@@ -98,6 +98,11 @@ func (b *Breaker) Maybe(ctx context.Context, thunk func()) bool {
 	}
 }
 
+// InFlight returns the number of requests currently in flight in this breaker.
+func (b *Breaker) InFlight() int {
+	return len(b.pendingRequests)
```

Comment:
neat, I didn't realize you could do that on a channel :)

Reply:
⭐️ 🌟 :)